tests/PHPStan/Type/ClosureTypeFactoryTest.php

```diff
@@ -0,0 +1,69 @@
+<?php declare(strict_types = 1);
+
+namespace PHPStan\Type;
+
+use Closure;
+use PHPStan\Testing\PHPStanTestCase;
+
+class ClosureTypeFactoryTest extends PHPStanTestCase
+{
+
+	public function dataFromClosureObjectReturnType(): array
+	{
+		return [
+			[static function (): void {
+			}, 'void'],
+			[static function () { // @phpcs:ignore
```

Comment:
Wanted to test a case where there is no native return type, but `phpcbf` was adding `void`. Needed to ignore it here.